releasenotes/notes/apm-update-meta-and-metrics-maximum-length-edc779e32cc6dcfe.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Move the maximum meta and metrics value length from 5k to 25k
```

> Increase the maximum size for span tag string values from 5k to 25k.

Official name is "span tag" (https://docs.datadoghq.com/tracing/visualization/#span-tags) and this impacts only meta (string), not metrics.

---

I'll change the release note but I'm not sure to understand your second comment.Metrics max length = meta max length (https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/traceutil/truncate.go#L27)

---

Agreed, will change accordingly

---

All trace-agent changes need to be prefixed with `APM: `.
```suggestion
    APM: The maximum allowed tag value length has been increased to 25,000 bytes.
```
Please rephrase it in a more user-friendly way.